blocks/how-to/src/app.tsx

```diff
@@ -271,217 +296,248 @@ export const App: BlockComponent<RootEntity> = ({
+                          variant="tertiary"
+                          size="small"
+                          sx={{ fontSize: 14 }}
+                          onClick={() => createIntroduction()}
+                          disabled={introButtonAnimatingOut}
+                        >
+                          <FontAwesomeIcon
+                            icon={{ icon: faPlus }}
+                            sx={{ mr: 1, fontSize: 13 }}
+                          />
+                          Add Introduction
+                        </Button>
+                      </Collapse>
+
+                      <Collapse
+                        in={!!introEntity && !introButtonAnimatingOut}
+                        onExited={() => setIntroAnimatingOut(false)}
+                        appear
+                      >
+                        <Step
+                          header="Introduction"
+                          title={introEntity?.properties[titleKey] ?? ""}
+                          titlePlaceholder="Requirements, Ingredients, Pre-requisites, etc."
+                          description={
+                            introEntity?.properties[descriptionKey] ?? ""
+                          }
+                          descriptionPlaceholder="Enter a list of things that might be helpful for people to know before they begin."
+                          updateField={updateIntroductionField}
+                          onRemove={() => removeIntroduction()}
+                          readonly={readonly}
+                          deleteButtonText="Remove intro"
+                        />
+                      </Collapse>
+                    </Box>
+                  ) : null}
+
+                  <Box>
+                    {steps.map((step, index) => (
+                      <Collapse
+                        key={step.id}
+                        in={!step.animatingOut}
+                        onExited={async () => {
+                          const newSteps = [...steps];
+                          newSteps.splice(index, 1);
+                          setSteps(newSteps);
+
+                          const stepLink =
+                            stepLinkedEntities[index]?.linkEntity;
+
+                          if (!stepLink) {
+                            return;
+                          }
+
+                          await graphModule.deleteEntity({
+                            data: {
+                              entityId: stepLink.metadata.recordId.entityId,
+                            },
+                          });
+                        }}
+                        appear
+                      >
+                        <Box
+                          sx={{
+                            mb: index === steps.length - 1 ? 0 : 3,
+                            transition: ({ transitions }) =>
+                              transitions.create("margin-bottom"),
+                          }}
+                        >
```

Comment:
Curious if this can be achieved via using  `display: flex` & `gap: 3` on the parent container instead of using `margin-bottom`? Or is the push down animation here something we want to keep?

Reply:
It does work to keep the spacing between steps, but it makes the delete step animation jump.